### ISSUE TITLE
oci_loggingsearch_search_fact shows NotImplementedError

### ISSUE THREAD

**Issue by halfbloodrahul** [oracle/oci-ansible-collection#228]
## Issue Report
I am trying to use oci_loggingsearch_search_fact module for performing a simple search query but it does not work
**Expected behavior**
I Should be getting a json response of  logs matching to the search query
**Environment**

* OS version:
macOS Montery 12.3
* Ansible version:
ansible [core 2.13.10]
  jinja version = 3.1.2
  libyaml = False

* OCI Python SDK version:
2.103.0

Collection Version
---------- -------
oracle.oci 4.23.0

**Ansible playbook to reproduce the issue**
```
- name: Get a Log
      oci_loggingsearch_search_facts:
        time_start: "2023-06-16T08:48:34Z"
        time_end: "2023-06-16T08:59:35Z"
        search_query: "search \"{{compartment_ocid}}\"| sort by datetime desc"
```
But Instead I get the following error:
```
The full traceback is:
Traceback (most recent call last):
  File "/Users/unk/.ansible/tmp/ansible-tmp-1613152.226-361236-2070150874/AnsiballZ_oci_loggingsearch_search_facts.py", line 107, in <module>
    _ansiballz_main()
  File "/Users/unk/.ansible/tmp/ansible-tmp-1613152.226-361236-2070150874/AnsiballZ_oci_loggingsearch_search_facts.py", line 99, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/Users/unk/.ansible/tmp/ansible-tmp-163152.226-361236-2070150874/AnsiballZ_oci_loggingsearch_search_facts.py", line 47, in invoke_module
    runpy.run_module(mod_name='ansible_collections.oracle.oci.plugins.modules.oci_loggingsearch_search_facts', init_globals=dict(_module_fqn='ansible_collections.oracle.oci.plugins.modules.oci_loggingsearch_search_facts', _modlib_path=modlib_path),
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/runpy.py", line 207, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/var/folders/r3/v9c9qdx52gq/T/ansible_oci_loggingsearch_search_facts_payload_oaanob1b/ansible_oci_loggingsearch_search_facts_payload.zip/ansible_collections/oracle/oci/plugins/modules/oci_loggingsearch_search_facts.py", line 102, in <module>
  File "/var/folders/r3/v9c9qdx52gq/T/ansible_oci_loggingsearch_search_facts_payload_oaanob1b/ansible_oci_loggingsearch_search_facts_payload.zip/ansible_collections/oracle/oci/plugins/modules/oci_loggingsearch_search_facts.py", line 96, in main
  File "/var/folders/r3/v9c9qdx52gq/T/ansible_oci_loggingsearch_search_facts_payload_oaanob1b/ansible_oci_loggingsearch_search_facts_payload.zip/ansible_collections/oracle/oci/plugins/module_utils/oci_resource_utils.py", line 183, in fail
  File "/var/folders/r3/v9c9qdx52gq/T/ansible_oci_loggingsearch_search_facts_payload_oaanob1b/ansible_oci_loggingsearch_search_facts_payload.zip/ansible_collections/oracle/oci/plugins/module_utils/oci_resource_utils.py", line 128, in get_required_params_for_list
NotImplementedError: list not supported by search
```

**Comment by dineshsuthar786** [comment]
@halfbloodrahul We have fixed the issue in latest release [v4.27.0](https://github.com/oracle/oci-ansible-collection/releases/tag/v4.27.0). Please check and let us know if you still face the issue.

**Comment by manojmeda** [comment]
@halfbloodrahul Closing the issue since it is fixed as @dineshsuthar786 mentioned. Please feel free to re-open or create a new ticket if you see any issues.
